packages/root/src/Root.story.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select } from '@storybook/addon-knobs';
+import Root from '.';
+
+storiesOf('Root', module)
+  .add('Anchor', () => (
+    <Root href="https://mongodb.design">
+      {text('Children', 'I am an anchor tag')}
+    </Root>
+  ))
+  .add('Custom', () => {
+    const CustomRoot = select(
+      'div',
+      { div: 'div', span: 'span', button: 'button' },
```

Comment:
I would specifically create or import a full component for use with this story (maybe `@leafygreen-ui/badge` since it is simple?) and offer `div`, `span`, and `Badge` instead. This would also avoid any confusion about needing to explicitly pass in `as="button"` since right now that's the default behavior.